test/jdk/java/awt/a11y/AccessibleTextTest.java

```diff
@@ -128,9 +128,18 @@ private void createTextArea() {
                 + "Check a11y of JTextArea.\n\n"
                 + "Turn screen reader on and press the arrow keys.\n\n"
                 + "If you can hear this instructions, tab further and press PASS, otherwise press FAIL.";
+        String value = """
+                apple
+                orange
+                mango
+                pineapple
+                :)
+                banana""";
 
         JPanel panel = new JPanel();
         panel.setLayout(new FlowLayout());
+        JTextArea textArea = new JTextArea(value);
+        panel.add(textArea);
```

Comment:
Hello. Thanks for the invitation.
This addition seems redundant to me. This is a test for a non-editable JTextArea. In it, the instruction is the same as the test.
If you press "Pass", the next test will appear, which has a multi-line text editor. This will also be a JTextArea, but already editable.

Reply:
Hi @savoptik, 
a) The test createTextArea() does not have JTextArea . I have attached the screen shot of the test https://bugs.openjdk.java.net/browse/JDK-8278828
b) The test does not say its a non-editable JTextArea. Even the exception String also say as follows
exceptionString = "Simple text area test failed!";  . 
c) If user press "Pass" to see the next test then there is no need of  createTextArea() . Correct me if I am wrong?

Reply:
a) The test createTextArea() does not have JTextArea .
YES, createTextArea () does not create a JTextArea, but the instruction field itself is not editable JTextArea. Take a look at the AccessibleComponentTest to be sure.
b) The test does not say its a non-editable JTextArea. Even the exception String also say as follows
exceptionString = "Simple text area test failed!"; .
Yes, the instructions do not indicate that this is not an editable JTextArea, you can supplement the instructions. As well as an exception String.
c) If user press "Pass" to see the next test then there is no need of createTextArea() .
There is a need.This test tries to cover all cases and possible configurations of various text components: single-line, multi-line, editable, non-editable - and various possible combinations. It is also convenient to use them when developing and debugging native implementations of a11y, for which they were created by me.

Reply:
I have changed the instruction.